src/main/java/com/imsweb/mph/DefaultHematoDbDataProvider.java

```diff
@@ -10,33 +10,32 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 import java.util.regex.Pattern;
-import java.util.stream.Collectors;
 
 import com.opencsv.CSVReaderBuilder;
 import com.opencsv.exceptions.CsvException;
 
-import com.imsweb.mph.MphUtils.MpResult;
 import com.imsweb.mph.internal.HematoDbDTO;
 
 /**
- * This is a default hemato db utils provider which uses seer-api to determine whether two morphologies are same primary, transform to or transform from according to
+ * This is a default hemato db data provider which uses seer-api to get same primary, transform to or transform from data from
  * hematopoietic and lymphoid neoplasm database.
  */
 @SuppressWarnings("java:S112") //This is a lab class and throwing RuntimeException is fine
-public class DefaultHematoDbUtilsProvider implements HematoDbUtilsProvider {
+public class DefaultHematoDbDataProvider implements HematoDbDataProvider {
```

Comment:
I am sorry to be a pain about this, but if we are going to rename this class, now is the time to remove that "Db" from everywhere. It just doesn't make sense. It was added 15 years ago because the very first standalone application IMS wrote for the Hematopoietic codes was called the "HematoDB" application. The name remains all those years. But that data has nothing to do with a database, and I find it confusing to keep that in the names.

Can you please rename the following:
HematoDbDataProvider to HematoDataProvider
HematoDbUtils to HematoUtils
HematoDbDTO to HematoData

I am not sure about that last one. IMO, the "data" is the entire data; the methods in the provider returns a list of individual DTO representing individual "disease". We shouldn't call them disease, but I feel like calling the individual elements "HematoData" is misleading as well.  At the, maybe it doesn't matter because we actually don't need a class that represents the entire data (it's always represented by lists). So I guess it's fine as-is. But if you find a better alternative that you like, feel free to change it (I was thinking "HematoEntry" or "HematoDataEntry", but I am not sure I like those better...

Please review the code for other places (like variable name) that would still reference the "Db" name (like the getHematoDbProvider() method in MphUtils for example.

Let me know if you don't agree with these changes and we can discuss.